main.c

```diff
@@ -1,78 +1,21 @@
 #include <stdio.h>
-#include <assert.h>
+#include "ColorPair.h"
 
-enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
-enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
-
-const char* MajorColorNames[] = {
-    "White", "Red", "Black", "Yellow", "Violet"
-};
-int numberOfMajorColors =
-    sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
-const char* MinorColorNames[] = {
-    "Blue", "Orange", "Green", "Brown", "Slate"
-};
+const char* MajorColorNames[] = {"White", "Red", "Black", "Yellow", "Violet"};
```

Comment:
`MajorColorNames` is defined here, while the `ColorCodes.h` has the codes of the colors. This split does not seem like an advantage.
Can we keep related things together?